include/yaclib/fault/detail/antagonist/yielder.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+// TODO(myannayx): define in cmake depending on system
+#define kSleepTime 500
```

Comment:
inline constexpr variable

Btw why in global namespace

Reply:
I thought that later it should be define that came from CMake so that's why it was SLEEP_TIME define before 🤔